plugins/woocommerce-admin/client/products/fields/attribute-field/add-attribute-modal.tsx

```diff
@@ -186,12 +210,15 @@ export const AddAttributeModal: React.FC< AddAttributeModalProps > = ( {
 																	'attributes[' +
 																		index +
 																		']',
-																	{
-																		...val,
-																		terms: [],
-																		options:
-																			undefined,
-																	}
+																	typeof val ===
```

Comment:
It would be great to have this as a util method so we could also test it.

E.g., `parseAttribute( val: string | Attribute )`

Reply:
Good suggestion, I did do this as part of the https://github.com/woocommerce/woocommerce/pull/35100/commits/fd8af0d7fe3a812dbff3f5ace8077e099f6298f8 commit (it includes the other changes as well)